tests/test_datasets/test_internal_datasets.py

```diff
@@ -73,13 +73,58 @@ def test_not_present_part():
         _ = load_dataset("custom_internal_dataset", parts="val")
 
 
-@pytest.mark.skip(reason="Dataset is too large for testing in GitHub.")
```

Comment:
Yo can make something like [this](https://doc.pytest.org/en/latest/how-to/skipping.html#skip-xfail-with-parametrize) instead of full removing